src/app/summary/mini-card/mini-card.component.css

```diff
@@ -0,0 +1,27 @@
+.mini-card-title {
+  color: #2A4B8C;
+  padding: 0px;
+  margin: 0px;
+  text-transform: capitalize;
+}
+.mini-card-value {
+  font-size: 4em;
+  color: #2A4B8C;
+  margin: 0px;
+  padding: 0px;
+}
+
+.mini-card {
+  box-shadow: 0 0.125rem 0.25rem rgba(84 88 90 50);
+  box-sizing: border-box;
+  -webkit-box-shadow: 0 .125rem .25rem rgba(84,88,90,.5);
```

Comment:
Just use box shadow. Webkit prefix is if you're only targetting chrome and safari. If we need prefixes to support older browsers then use [autoprefixer](https://github.com/postcss/autoprefixer)

Reply:
Id did not work without the -webkit-box-shadow. Not sure what I am missing.